content/ja/handbook/finance/financial-planning-and-analysis/index.md

```
@@ -0,0 +1,614 @@
+
+1. GitLabの戦略に沿った部門横断的な実行を促進する
+1. GitLabに予測可能性と運営の厳格さをもたらす
+1. GitLabの財務目標・運営目標が定義・文書化・達成されることを確保する
+1. 戦略と各部門がそれを達成するための役割についての認知を広める
+1. 健全なデータドリブンな意思決定サポートを確保する
+1. 私たちの公開企業としての対外的なメッセージが運営戦略と一致していることを確保する
+
+## GitLabのFP&Aが目標を達成するための方法
+
+1. GitLabの取締役会計画の予算・計画プロセスを管理する
+1. 長期戦略と財務目標を特定する[長期見通し（LRO）](#long-range-outlook-lro)を構築・維持する
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)